### PR TITLE
fix #610 change the css style to the css in js

### PR DESCRIPTION
**Pull request by Aria622**
delete the css in style file
move the css style to the css in js

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,6 +9,9 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@emotion/react": "^11.11.4",
+    "@emotion/styled": "^11.11.5",
+    "antd": "^5.19.0",
```
Comment (ucfyao) [inline package.json:14]: We do not use antd, please do not download it.

File: src/app/newmarket/page.tsx [hunk 2]
```diff
@@ -79,27 +81,88 @@ const NewmarketForm: React.FC = () => {
...
+  margin-top: 8px;
+  font-size: 0.9em;
+`;
+  const imgStyle = css`
+  width: 22px;
+  margin-right: 5px;
+`;
+
+
   return (
```
Comment (ucfyao) [inline src/app/newmarket/page.tsx:142]: You can use a variable to contain all CSS, just like before.

File: src/app/newmarket/page.tsx [hunk 3]
```diff
@@ -79,27 +81,88 @@ const NewmarketForm: React.FC = () => {
     return null;
   }
 
+   const styleBox = css`
+  margin: 0 auto;
+  padding: 50px;
+  background-color: white;
+  border-radius: 8px;
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+`;
```
Comment (Yingxue0323) [inline src/app/newmarket/page.tsx:90]: please fix the indentation(including this one and every one follows)

File: src/app/newmarket/page.tsx [hunk 4]
```diff
@@ -79,27 +81,88 @@ const NewmarketForm: React.FC = () => {
...
+  margin-right: 5px;
+`;
+
+
   return (
     <RootLayout>
       <div className="container mx-auto" style={{ marginTop: '70px', maxWidth: '1200px' }}>
         <section className="section">
-          <div className="box">
+          <div css={styleBox}>
```
Comment (Yingxue0323) [inline src/app/newmarket/page.tsx:146]: Include all previous css in a variable named maybe 'style'. This way, you can use {style.box} to access it here, and later you can use {style.choice}, and so on.

File: package.json (hunk 1 above)
Comment (Aria622) [reply package.json]: done

File: src/app/newmarket/page.tsx (hunk 2 above)
Comment (Aria622) [reply src/app/newmarket/page.tsx]: done

File: src/app/newmarket/page.tsx (hunk 3 above)
Comment (Aria622) [reply src/app/newmarket/page.tsx]: done

File: src/app/newmarket/page.tsx (hunk 4 above)
Comment (Aria622) [reply src/app/newmarket/page.tsx]: done

**Review by ucfyao — APPROVED** [review]

**Pull request merged**